CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required (VERSION 3.1)
```

Comment:
I think we could be a little more verbose on the content of the cmakelists as ti could grow more, if we integrate more functionality to this software